tests/run_vector_tests.sh

```diff
@@ -9,18 +9,22 @@ fi
 
 pip install -r "$1/requirements.txt"
 $1/generator.py -t uptane --signature-encoding base64 -o vectors --cjson json-subset
-$1/server.py -t uptane --signature-encoding base64 &
+if [ "$2" == "valgrind" ]; then
+  $1/server.py -t uptane --signature-encoding base64 -P 8081 &
```

Comment:
@patriotyk Not a bad idea, but that gets a bit challenging in this case. We need to generate the port in this script so that we can pass it to both the python server and the C++ test case.